01_install_requirements.sh

```diff
@@ -56,6 +56,9 @@ popd
 # Needed to get a recent python-virtualbmc package
 sudo tripleo-repos current-tripleo
 
+# Disable rdo-qemu-ev repo as there are issues running in nested virt currently
+sudo yum-config-manager --disable rdo-qemu-ev
```

Comment:
Do you have any more info on what issues you hit when this repo isn't disabled?

Reply:
will come back with some more concrete logs.

Reply:
OK, thanks.

Reply:
will dig deeper but:

```
[root@pete-centos-1 pete]# /usr/libexec/qemu-kvm --version
QEMU emulator version 2.12.0 (qemu-kvm-ev-2.12.0-18.el7_6.3.1)
Copyright (c) 2003-2017 Fabrice Bellard and the QEMU Project developers
```

```
May 13 21:28:17 pete-centos-1 systemd[1]: Started Virtualization daemon.
May 13 21:28:17 pete-centos-1 dnsmasq[58415]: read /etc/hosts - 2 addresses
May 13 21:28:17 pete-centos-1 dnsmasq[58235]: read /etc/hosts - 2 addresses
May 13 21:28:17 pete-centos-1 dnsmasq[58415]: read /var/lib/libvirt/dnsmasq/baremetal.addnhosts - 2 addresses
May 13 21:28:17 pete-centos-1 dnsmasq[58235]: read /var/lib/libvirt/dnsmasq/default.addnhosts - 0 addresses
May 13 21:28:17 pete-centos-1 dnsmasq-dhcp[58235]: read /var/lib/libvirt/dnsmasq/default.hostsfile
May 13 21:28:17 pete-centos-1 dnsmasq-dhcp[58415]: read /var/lib/libvirt/dnsmasq/baremetal.hostsfile
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: started, version 2.76 cachesize 150
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: compile time options: IPv6 GNU-getopt DBus no-i18n IDN DHCP DHCPv6 no-Lua TFTP no-conntrack ipset auth no-DNSSEC loop-detect inotify
May 13 21:28:23 pete-centos-1 dnsmasq-dhcp[60232]: DHCP, IP range 192.168.111.20 -- 192.168.111.60, lease time 1h
May 13 21:28:23 pete-centos-1 dnsmasq-dhcp[60232]: DHCP, sockets bound exclusively to interface baremetal
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: using local addresses only for domain ostest.test.metalkube.org
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: using nameserver 127.0.0.1#53 for domain apps.ostest.test.metalkube.org
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: reading /etc/resolv.conf
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: using local addresses only for domain ostest.test.metalkube.org
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: using nameserver 127.0.0.1#53 for domain apps.ostest.test.metalkube.org
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: using nameserver 168.63.129.16#53
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: read /etc/hosts - 2 addresses
May 13 21:28:23 pete-centos-1 dnsmasq[60232]: read /var/lib/libvirt/dnsmasq/baremetal.addnhosts - 2 addresses
May 13 21:28:23 pete-centos-1 dnsmasq-dhcp[60232]: read /var/lib/libvirt/dnsmasq/baremetal.hostsfile
May 13 21:28:24 pete-centos-1 dnsmasq[58235]: reading /etc/resolv.conf
May 13 21:28:24 pete-centos-1 dnsmasq[60232]: reading /etc/resolv.conf
May 13 21:28:24 pete-centos-1 dnsmasq[60232]: using local addresses only for domain ostest.test.metalkube.org
May 13 21:28:24 pete-centos-1 dnsmasq[58235]: using nameserver 127.0.0.1#53
May 13 21:28:24 pete-centos-1 dnsmasq[60232]: using nameserver 127.0.0.1#53 for domain apps.ostest.test.metalkube.org
May 13 21:28:24 pete-centos-1 dnsmasq[60232]: using nameserver 127.0.0.1#53
May 13 21:40:58 pete-centos-1 dnsmasq[63744]: started, version 2.76 DNS disabled
May 13 21:40:58 pete-centos-1 dnsmasq[63744]: compile time options: IPv6 GNU-getopt DBus no-i18n IDN DHCP DHCPv6 no-Lua TFTP no-conntrack ipset auth no-DNSSEC loop-detect inotify
May 13 21:40:58 pete-centos-1 dnsmasq-dhcp[63744]: DHCP, IP range 192.168.39.2 -- 192.168.39.254, lease time 1h
May 13 21:40:58 pete-centos-1 dnsmasq-dhcp[63744]: DHCP, sockets bound exclusively to interface virbr1
May 13 21:40:58 pete-centos-1 dnsmasq-dhcp[63744]: read /var/lib/libvirt/dnsmasq/minikube-net.hostsfile
May 13 21:40:59 pete-centos-1 libvirtd[59734]: 2019-05-13 21:40:59.510+0000: 59737: info : libvirt version: 4.5.0, package: 10.el7_6.7 (CentOS BuildSystem <http://bugs.centos.org>, 2019-04-24-14:04:12, x86-01.bsys.centos.org)
May 13 21:40:59 pete-centos-1 libvirtd[59734]: 2019-05-13 21:40:59.510+0000: 59737: info : hostname: pete-centos-1
May 13 21:40:59 pete-centos-1 libvirtd[59734]: 2019-05-13 21:40:59.510+0000: 59737: warning : qemuDomainObjTaint:7521 : Domain id=1 name='minikube' uuid=3768d2f1-506f-4b25-b285-1cf08aa639d0 is tainted: host-cpu
May 13 21:43:13 pete-centos-1 libvirtd[59734]: 2019-05-13 21:43:13.264+0000: 59734: error : virNetSocketReadWire:1806 : End of file while reading data: Input/output error
May 13 21:43:13 pete-centos-1 libvirtd[59734]: 2019-05-13 21:43:13.265+0000: 59734: error : virNetSocketReadWire:1806 : End of file while reading data: Input/output error
```

```
[root@pete-centos-1 pete]# cat /proc/cpuinfo
processor	: 0
vendor_id	: GenuineIntel
cpu family	: 6
model		: 79
model name	: Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
stepping	: 1
microcode	: 0xffffffff
cpu MHz		: 2294.688
cache size	: 51200 KB
physical id	: 0
siblings	: 32
core id		: 0
cpu cores	: 16
apicid		: 0
initial apicid	: 0
fpu		: yes
fpu_exception	: yes
cpuid level	: 20
wp		: yes
flags		: fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ss ht syscall nx pdpe1gb rdtscp lm constant_tsc rep_good nopl xtopology eagerfpu pni pclmulqdq vmx ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm abm 3dnowprefetch tpr_shadow vnmi ept vpid fsgsbase bmi1 hle avx2 smep bmi2 erms invpcid rtm rdseed adx smap xsaveopt
bogomips	: 4589.37
clflush size	: 64
cache_alignment	: 64
address sizes	: 44 bits physical, 48 bits virtual
power management:
```

Reply:
OK, I'll try to get an env where I can replicate the issue as well